doc/ArchInstall/arch-chroot_install.sh

```diff
@@ -7,14 +7,18 @@ HOME="/home/${USER}"
 HOST="nugus"
 HOSTNAME="${HOST}${ROBOT_NUMBER}"
 IP_ADDR="10.1.1.${ROBOT_NUMBER}"
+ETHERNET_INTERFACE="${ETHERNET_INTERFACE:-eth0}"
+WIFI_INTERFACE="${WIFI_INTERFACE:-wlan0}"
```

Comment:
Is there a way to enforce that these go to the hardware based names rather than these names?